test/struct.test.js

```diff
@@ -405,3 +416,171 @@ describe("Cip68 Pair[Int, Int]", () => {
         })
     })
 })
+
+// this space intentionally left blank.  See below.
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//hack hack hack.
```

Comment:
ha, safe to remove.  

Was experimenting with the notion of possibly encoding line-numbers as hints in generated or inline-defined sources such as these, which would allow source-mapping for these cases as well as for standalone .hl files.